app.rb

```diff
@@ -0,0 +1,30 @@
+require 'sinatra/base'
+require './lib/message.rb'
+require './lib/account.rb'
+require 'date'
+require './lib/database_connection_setup.rb'
+
+class Chitter < Sinatra::Base
```

Comment:
Think about taking a look at RESTful routing. It's a convention for the way we name routes in a web application and a pretty east win for making your code look of a professional standard.

Reply:
Other than that, this controller looks good.  Your routes are pretty concise and aren't responsible for any heavy logical lifting.